src/mpu9250_interface.cpp

```diff
@@ -8,7 +8,7 @@
 #include "mpu9250_interface.hpp"
 #include "memory_map.hpp"
 
-MPU9250Interface::MPU9250Interface(hwlib::pin_oc &scl, hwlib::pin_oc &sda, const int8_t MPUAddr)
+MPU9250Interface::MPU9250Interface(hwlib::pin_oc &scl, hwlib::pin_oc &sda, const uint8_t MPUAddr)
     : scl(scl), sda(sda), MPUAddr(MPUAddr), i2c(hwlib::i2c_bus_bit_banged_scl_sda(scl, sda)) {
 
     hwlib::cout << hwlib::right << "Initializing MPU9250..." << hwlib::endl;
```

Comment:
I gues this is only usefull for debuging and should not be in the final product. Or is there a reason for this print statement except for debugging?